index.html

```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Slide Show</title>
+		<style>
+		body{
+
+		}
+		.slideShow-container{
+			background-color: rgba(255,255,255,0.3);
+			position: absolute;
+			height: 95%;
+			width: 100%;
+			top: 10%;
+			left: 0%;
+		}
+		.slides-container{
+			position: absolute;
+			height: 100%;
+    		width: 100%;
+    		top: 0;
+    		left: 0;
+
+		}
+		.btn{
+			background-repeat: no-repeat;
+    		position: absolute;
+    		top: 0;
+    		height: 100%;
+    		width: 4%;
+    		cursor: pointer;
+    		opacity: 0.3;
+		}
+		.Option-bar{
+			background-color: rgba(0,0,0,0.5);
+			position: absolute;
+			top: 94%;
+			height: 6%;
+			width: 100%;
+		}
+		.play{
+			left: 75%;
+			background-image: url(https://png.icons8.com/nolan/50/000000/circled-play.png);
+		}
+		.prev{
+			left: 65%;
+			background-image: url(https://png.icons8.com/nolan/50/000000/back.png);
+		}
+		.next{
+			left: 85%;
+			background-image: url(https://png.icons8.com/nolan/50/000000/forward.png);
+		}
+		.thumbnails{
+			left: 23.5%;
+			background-image: url(https://png.icons8.com/nolan/50/000000/stack-of-photos.png);
+		}
+		.info{
+			left: 10%;
+			background-image: url(https://png.icons8.com/nolan/50/000000/plus-math.png);
+		}
+		.counter{
+    		position: absolute;
+    		top: 0;
+    		height: 100%;
+    		width: 10%;
+    		left: 45%;
+    		text-align: center;
+			margin: auto;
+			height: 100%;
+			font-size: 255%;
+		}
+		.imgs{
+			position: absolute;
+			height: 100%;
+			overflow: hidden;
+			visibility: hidden;
+			width: 100%
+		}
+		.imgs img{
```

Comment:
why use `.imgs img` or maybe you could use direct child selector `>` to be more precise

Reply:
I was thinking on using it, but didn't remember at the moment and the forget to do it.